File: frontend/src/app/doktor/doktorGetAllResponse.ts
export interface DoktorGetAllResponse{
    doktori:DoktorGetAllResponseDoktor[]
}
export interface DoktorGetAllResponseDoktor {
    zaposlenikId: number
    imePrezime: string
    jmbg: string
    datumRodjenja: string
    datumZaposlenja: string
    nalogId: any
    poslovnaPozicijaId: number
    nazivKlinike: string
    oblastMedicine: string
    specijalizacija: string
}

File: DomZaStaraLicaApi/Endpoints/Doktor/GetAllDoktor/DoktorGetAllResponse.cs
using DomZaStaraLicaApi.Endpoints.Njegovatelj.GetAllNjegovatelji;

namespace DomZaStaraLicaApi.Endpoints.Doktor.GetAllDoktor
{
    public class DoktorGetAllResponse
    {
        public List<DoktorGetAllResponseDoktor> Doktori { get; set; }
    }

    public class DoktorGetAllResponseDoktor
    {
        public int ZaposlenikId { get; set; }
        public string ImePrezime { get; set; }
        public string JMBG { get; set; }
        public DateTime DatumRodjenja { get; set; }
        public DateTime DatumZaposlenja { get; set; }
        public int? NalogId { get; set; }
        public int PoslovnaPozicijaId { get; set; }
        public string NazivKlinike { get; set; }
        public string OblastMedicine { get; set; }
        public string Specijalizacija { get; set; }
    }
}
